app/src/call_types.h

```diff
@@ -87,55 +71,22 @@ namespace scitt
   DECLARE_JSON_TYPE(GetVersion::Out);
   DECLARE_JSON_REQUIRED_FIELDS(GetVersion::Out, version);
 
-  struct GetEntry
-  {
-    struct Out
-    {
-      ccf::TxID entry_id;
-    };
-  };
-
-  DECLARE_JSON_TYPE(GetEntry::Out);
-  DECLARE_JSON_REQUIRED_FIELDS_WITH_RENAMES(GetEntry::Out, entry_id, "entryId");
-
   struct GetOperation
   {
     struct Out
     {
       ccf::TxID operation_id;
       OperationStatus status;
       std::optional<ccf::TxID> entry_id;
+      // TODO: it needs to be in expcted structure as per RFC
       std::optional<nlohmann::json> error;
```

Comment:
@achamayou Is there some sane way to express this error in a structure which will be then converted to cbor?

At the moment the implementation does create a struct then converts to json then json to cbor:
```
operation = {
.operation_id = operation_id,
.status = OperationStatus::Failed,
.error =
  ODataError{
    .code = ccf::errors::TransactionInvalid,
    .message = "Transaction is invalid",
  },
};
...
auto body = nlohmann::json(operation);
ctx.rpc_ctx->set_response_body(nlohmann::json::to_cbor(body));
```